gnocchi/storage/__init__.py

```diff
@@ -717,3 +725,9 @@ def _map_compute_splits_operations(bound_timeserie):
         with self.statistics.time("raw measures store"):
             self._store_unaggregated_timeseries(new_boundts)
         self.statistics["raw measures store"] += len(new_boundts)
+
+    def get_latest_timestmap_of_measures(self, measures):
+        oldest_timestamp_in_measurements = measures['timestamps'][-1]
```

Comment:
Are you sure that its always sorted? maybe using max(measures['timestamps']) would be something more solid.

Reply:
agree, done.